flash_examples/speech_recognition.py

```diff
@@ -29,16 +29,17 @@
 )
 
 # 2. Build the task
-model = SpeechRecognition(backbone="facebook/wav2vec2-base-960h")
+model = SpeechRecognition(backbone="facebook/wav2vec2-base-960h", learning_rate=1e-5)
 
 # 3. Create the trainer and finetune the model
 trainer = flash.Trainer(max_epochs=1, gpus=torch.cuda.device_count())
 trainer.finetune(model, datamodule=datamodule, strategy="no_freeze")
 
 # 4. Predict on audio files!
-datamodule = SpeechRecognitionData.from_files(predict_files=["data/timit/example.wav"], batch_size=4)
-predictions = trainer.predict(model, datamodule=datamodule)
-print(predictions)
+predict_datamodule = SpeechRecognitionData.from_files(predict_files=["data/timit/example.wav"], batch_size=4)
+predictions = trainer.predict(model, datamodule=predict_datamodule)
```

Comment:
Let's keep the name datamodule for now just to be consistent with the other examples:
```suggestion
datamodule = SpeechRecognitionData.from_files(predict_files=["data/timit/example.wav"], batch_size=4)
predictions = trainer.predict(model, datamodule=datamodule)
```